### PR TITLE
Feat: added Login and Signup pages

### PR DESCRIPTION
**Pull request by kjais1720**
Added Login and Signup page
https://user-images.githubusercontent.com/66024105/154333469-cdde4b5b-d700-4be7-bd6b-9c33828148cc.mp4

### CODE REVIEW

File: pages/auth.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | Cickstock</title>
+    <link rel="shortcut icon" href="../assets/favicon.jpeg" type="image/x-icon">
+    <link rel="stylesheet" href="https://tarang-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../styesheets/base.css">
```
Comment (shuntzhere) [inline pages/auth.html]: Can this component library be moved to the base.css file? Since you will be using that on every HTML page.

**Review by shuntzhere — APPROVED** [review]
HTML is very clean with nice use of utility classes throughout the file. Nice work.

File: pages/auth.html (hunk 1 above)
Comment (kjais1720) [reply pages/auth.html]: Thanks for pointing it out, fixed in the latest commit

File: pages/auth.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | Cickstock</title>
```
Comment (kalyandindu) [inline pages/auth.html]: Crickstock, Miss spelled I think

**Review by kalyandindu — APPROVED** [review]
Code looks clean and good to me.

File: pages/auth.html (hunk 2 above)
Comment (kjais1720) [reply pages/auth.html]: Thanks for pointing it out, I have fixed it in the latest commit

**Pull request merged**